course3/course_en.md

```diff
@@ -18,19 +18,6 @@ style: |
 <!--
 ```moonbit
 let pi = 3.1415
-
-fn put(map: @map.Map[Int, Int64], num: Int, result: Int64) -> @map.Map[Int, Int64] {
-  map.insert(num, result)
-}
-
-fn get(map: @map.Map[Int, Int64], num: Int) -> Option[Int64] {
-  map.lookup(num)
-}
-
-fn make() -> @map.Map[Int, Int64] {
-  @map.empty()
-}
-
```

Comment:
I will use `insert`, `lookup` and `empty` directly in the examples.

Reply:
I would prefer using the abstract interface here. The reason is that the core is not stable yet.

Reply:
However, doing so will make the interfaces defined in P37 inconsistent with the interfaces we actually use later. Maybe we need to shed some light on this.

Reply:
Then I will also use `make`, `put`, `get` instead of `empty`, `insert`, `lookup` in P37.